src/test/java/com/commercetools/sync/inventories/utils/InventoryReferenceReplacementUtilsTest.java

```diff
@@ -55,24 +65,74 @@
         replaceInventoriesReferenceIdsWithKeys_WithNonExpandedReferences_ShouldReturnReferencesWithoutReplacedKeys() {
         final String customTypeId = UUID.randomUUID().toString();
         final List<InventoryEntry> mockInventoryEntries = new ArrayList<>();
+
```

Comment:
preparation/test/assertion phases missing.

Reply:
yep, it's actually something new that I am adding to the newly added tests because I think it makes reading the tests much more easier. But I cannot added to all the tests that were already existing.

Reply:
@lojzatran 5802457